src/rules/no-large-snapshots.ts

```diff
@@ -125,7 +132,10 @@ export default createRule<[RuleOptions], MessageId>({
                 'toThrowErrorMatchingInlineSnapshot',
               ))
           ) {
-            reportOnViolation(context, node, options);
+            reportOnViolation(context, node, {
+              ...options,
+              maxSize: options.inlineMaxSize,
```

Comment:
These something not quite right here: this will mean that if I don't pass `inlineMaxSize`, it'll use the default of `50` rather than `maxSize`.

So this should be:

```suggestion
              maxSize: options.inlineMaxSize ?? options.maxSize,
```

*However*, it seems that the tests don't cover this. Could you look into that for me, and see what's up?
I suspect that committing this will make coverage reveal the lack of ... well, coverage 😅 

(I suspect we're missing a test where we don't have `inlineMaxSize`, `maxSize` set greater than 50, and an inline snapshot that's greater than 50)

Reply:
Done, I have added a test to this scenario.